model/time.go

```diff
@@ -186,6 +186,10 @@ var durationRE = regexp.MustCompile("^([0-9]+)(y|w|d|h|m|s|ms)$")
 // ParseDuration parses a string into a time.Duration, assuming that a year
 // always has 365d, a week always has 7d, and a day always has 24h.
 func ParseDuration(durationStr string) (Duration, error) {
+	// shortcut if the value is 0
```

Comment:
This implies that this is a fast path, which it isn't.

Reply:
Why isn't it a fast path? If I don't have to spin up the regexp machinery, I'd totally call it a fast path.

In different news: Should we accept `"-0"`? https://golang.org/pkg/time/#ParseDuration does, BTW…

In yet different news, comment should start capitalized and end with a period.

Reply:
I guess what you might refer to is that "0" would not pass the code below, and thus it's not only fast, but _also_ handling a special case. (In general, explaining your thoughts a tad more detailed might help…)

Reply:
I'll improve this comment should we also support "-0" ? You tell me.

Reply:
Thinking about it: We do not support negative durations in general, so we should not support `"-0"` either.